CompileTests/MultiModule/Sources/ModuleA/a.pb.swift

```diff
@@ -103,18 +103,14 @@ public let Extensions_ext_str = SwiftProtobuf.MessageExtension<SwiftProtobuf.Opt
 // MARK: - Code below here is support for the SwiftProtobuf runtime.
 
 extension E: SwiftProtobuf._ProtoNameProviding {
-  public static let _protobuf_nameMap: SwiftProtobuf._NameMap = [
-    0: .same(proto: "E_UNSET"),
-    1: .same(proto: "E_A"),
-    2: .same(proto: "E_B"),
-  ]
+  private static let _protobuf_nameMap_bytecode: Swift.StaticString = "\0\u{2}\0E_UNSET\0\u{1}E_A\0\u{1}E_B\0"
```

Comment:
Seems this could be encoded as `\0\u{1}E_UNSET\0\u{1}E_A\0...`. That is, if the first field is zero, shouldn't we be able to use `sameNext` ?

Reply:
The counter starts at 0, so using `sameNext` on the first value would pre-increment it and set `E_UNSET` to 1. So this works perfectly for field numbers that typically start at 1, but it's an awkward edge case for many enums.

We could resolve it by adding new `sameZero` and `aliasesZero` opcodes (since those are the only naming schemes used by enums). I didn't do it because it's additional runtime complexity for exactly one byte per enum, but I'm not opposed to making the change either.

Reply:
Ah.  Yes, that makes sense.  Good as-is then.